src/algebra/star/self_adjoint.lean

```diff
@@ -16,15 +16,6 @@ operators on Hilbert spaces.
 
 ## TODO
 
-* If `R` is a `star_module R₂ R`, put a module structure on `self_adjoint R`. This would naturally
-  be a `module (self_adjoint R₂) (self_adjoint R)`, but doing this literally would be undesirable
-  since in the main case of interest (`R₂ = ℂ`) we want `module ℝ (self_adjoint R)` and not
-  `module (self_adjoint ℂ) (self_adjoint R)`. One way of doing this would be to add the typeclass
-  `[has_trivial_star R]`, of which `ℝ` would be an instance, and then add a
-  `[module R (self_adjoint E)]` instance whenever we have `[module R E] [has_trivial_star E]`.
-  Another one would be to define a `[star_invariant_scalars R E]` to express the fact that
```

Comment:
Why did you decide for the first option instead of the second? It might be good to record the reasoning somewhere.

Reply:
The only reason is that with this solution, the new typeclass only takes one type as argument. Otherwise I think they're basically equivalent. I added an implementation note in the docstring explaining this.